tests/GithubIntegration.py

```diff
@@ -67,6 +56,15 @@ def testDeprecatedAppAuth(self):
             "instead",
         )
 
+    def testRequiredAppAuth(self):
+        # GithubIntegration requires AppAuth authentication.
+        for auth in [self.oauth_token, self.jwt, self.login]:
+            with self.assertRaisesRegex(
+                AssertionError,
+                "GithubIntegration requires github.Auth.AppAuth authentication, not .*",
+            ):
+                github.GithubIntegration(auth=self.oauth_token)
```

Comment:
```suggestion
            with self.assertRaises(
                AssertionError,
                f"GithubIntegration requires github.Auth.AppAuth authentication, not {type(auth)}",
            ):
                github.GithubIntegration(auth=auth)
```

Reply:
good catch on the auth vs self.oauth_token :)